system/config/app.php

```diff
@@ -15,6 +15,7 @@
         'salt' => FALSE,
         'httponly' => FALSE,
         'secure' => FALSE,
+        'domain' => NULL,
```

Comment:
need change cookie domain NULL to some string, maybe empty

Reply:
Yes, you have to set cookie domain in your project before start